documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1892,6 +1892,18 @@ See the {link-prefix}:{link-postgresql-connector}#postgresql-data-types[list of
 Each item (regular expression) must match the fully-qualified `<fully-qualified table>:<a comma-separated list of columns>` representing the custom key. +
 Fully-qualified tables could be defined as _schemaName_._tableName_.
 
+|[[postgresql-publication-autocreate-mode]]<<postgresql-publication-autocreate-mode, `publication.autocreate.mode`>>
+|_all_tables_
+|Determine how creation of a publication should work, the default is all_tables.
+_disabled_ - The connector will not attempt to create a publication at all. The expectation is
+that the user has created the publication up-front. If the publication isn't found to exist upon
+startup, the connector will throw an exception and stop.
+_all_tables_ - This acts like the current behavior where if the publication does not exist the
```

Comment:
Let's avoid the notion of "current" behavior, as it won't be clear to the future reader:

> If no publication exists, the connector will create a new publication for all tables. Note this requires xyz grants. If the publication already exists, it will be used.